packages/app-store/_components/AppCard.tsx

```diff
@@ -16,24 +16,22 @@ export default function AppCard({
   switchOnClick,
   switchChecked,
   children,
-  setAppData,
   returnTo,
   teamId,
-  disableSwitch,
-  LockedIcon,
+  appDataSchema,
```

Comment:
We do need to pass the appDataSchema though so the context gets the right types.

https://github.com/calcom/cal.com/pull/9950#discussion_r1253492034